data-generation/src/test/scala/SnapshotLandySpec.scala

```diff
@@ -83,4 +81,23 @@ class SnapshotLandySpec extends FixtureAnyFlatSpec with SparkTestWrapper {
       landyEdges.collect().sorted.zip(deltaEdges.collect().sorted).foreach({ case (e1, e2) => assert(e1 == e2) })
     })
   }
+
+  "directNeighbours" should "be equal for Landy and SnapshotDelta" in { f =>
+    val FixtureParam(_, entities, landyGraph, snapshotDeltaGraph, _) = f
+    val vertices = entities.flatMap(e => e match {
+      case v: VERTEX => Some(v)
+      case _: EDGE => None
+    })
```

Comment:
```suggestion
    val vertices = entities.flatMap(entity => entity match {
      case v: VERTEX => Some(v)
      case _: EDGE => None
    })
```
For å ikke forveksle med edge

Reply:
ja, den er litt wops